tests/virt_autotest/login_console.pm

```diff
@@ -157,5 +157,19 @@ sub run {
     $self->login_to_console;
 }
 
+sub post_fail_hook {
+    my ($self) = @_;
+    if (check_var('PERF_KERNEL', '1')) {
+        select_console 'log-console';
+        save_screenshot;
+        script_run "save_y2logs /tmp/y2logs_clone.tar.bz2";
```

Comment:
This log name is a little bit confusing for others with clone. Generally y2logs collected in other modules is just named as y2logs.tar.bz2, so suggest to follow general log name.

Reply:
ok,updated.